tools/Dashboard/DevHome.Dashboard/Helpers/WidgetIconCache.cs

```diff
@@ -15,6 +14,8 @@
 namespace DevHome.Dashboard.Helpers;
 internal class WidgetIconCache
 {
+    private static DispatcherQueue _dispatcher;

```

Comment:
Why is this static?

Reply:
I see. I suppose the proper solution would be to fully switch the codebase to use a single service provider that's injected all the way through, and that controls can retrieve from just the current application instance, but that'd also be more work and outside of the scope of this PR. Thank you for clarifying though! 🙂

FWIW: I would suggest evaluating doing such a switch especially while the codebase is still relatively small. We're gradually migrating to this from the previous "everything's static lol" approach that the Microsoft Store has been using since it's early days, and there's tons of benefits (especially catching bugs early), but man it's a _lot_ of work. I certainly wish this effort had been started much sooner, possibly from day 1. If this is something folks are considering doing with Dev Home eventually, I'd suggest not putting this off for too long, as it'd save a fair amount of time and effort 😅

Reply:
I totally agree! I even started picking away at doing some refactoring with #1221, but obviously there's a long way to go, and I should be moving Dashboard in the service direction.